app/components/image/image.html

```diff
@@ -59,7 +59,11 @@
 <div class="row">
   <div class="col-lg-12 col-md-12 col-xs-12">
     <rd-widget>
-      <rd-widget-header icon="fa-tag" title="Tag the image"></rd-widget-header>
+      <rd-widget-header icon="fa-tag" title="Tag the image">
+        <div class="pull-right">
```

Comment:
Please keep the buttons in the panel where needed as this is out of scope and discussed at the moment in #1408